test/ShellTests.cmake

```diff
@@ -15,8 +15,39 @@
 ###           T E S T I N G  S H E L L  S C R I P T S                      ###
 ##############################################################################
 
-if (UNIX)
-
+find_program (PWSH NAMES pwsh powershell)
```

Comment:
Does find_program need to run multiple times? It's also in test/CMakeTests.cmake above.

Reply:
Will remove from parent.